src/Microsoft.Diagnostics.EventFlow.Core/Implementations/DiagnosticPipelineFactory.cs

```diff
@@ -316,25 +316,25 @@ private static void ReportInvalidPipelineConfiguration(IHealthReporter healthRep
             Debug.Assert(healthReporter != null);
 
             inputFactories = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
-            inputFactories["EventSource"] = "Microsoft.Diagnostics.EventFlow.Inputs.EventSourceInputFactory, Microsoft.Diagnostics.EventFlow.Inputs.EventSource, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a";
```

Comment:
These looks like real-signed public token. I believe these are going to be used in the reflector for loading. Are these types still going to be loaded in signed bits in run-time?

Reply:
Yes. The signing that remains is Authenticode signing that confirms the DLL was build by Microsoft; it does not affect the .NET assembly loader.

Reply:
Okay. I don't think I understand the auth-code process but I'll trust your words on this. 💯